x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -729,6 +747,120 @@ export class ActionsClient {
   public isPreconfigured(connectorId: string): boolean {
     return !!this.preconfiguredActions.find((preconfigured) => preconfigured.id === connectorId);
   }
+
+  public async getGlobalExecutionLogWithAuth({
```

Comment:
can we add unit tests for these new functions inside `actions_client.test.ts`?

Reply:
yes, I can add those!

Reply:
Resolved in this commit 6bc2b5c56283d6357d3e66186c8afceaea582e4f